tests/ballerina-unit-test/src/test/resources/test-src/streaming/alias/streamingv2-filter-test.bal

```diff
@@ -68,7 +68,6 @@ function startFilterQuery() returns (Teacher[]) {
             break;
         }
     }
-    io:println(globalEmployeeArray);
```

Comment:
Shall we remove the unused import to ballerina/io in all the files?

Reply:
yes, will do that